### PR TITLE
Improve searching accounts with Pleroma/MastoFE

### PR DESCRIPTION
**Pull request by amyspark**
I've been testing Pleroma in the last couple of days, and Whalebird's search functionality does not work reliably with it.
First, the canonical URL of Pleroma users is of the form `https://domain.tld/users/username`, which is not expected by Whalebird. 
Pleroma's MastoFE API also doesn't work as expected: the most reliable way to find an account is to query it with the user's canonical URL, and then filter the results by either:
 - `username@domain.tld` (remote accounts)
 - by testing the canonical URL of each result (local accounts do not include the domain in their `acct` field).

I've tested these changes in mastodon.social and pleroma.site and they work as expected.

### CODE REVIEW

File: src/renderer/utils/tootParser.js [hunk 1]
```diff
@@ -71,7 +76,8 @@ export function parsePleromaAccount (accountURL) {
   const domainName = res[1]
   const accountName = res[2]
   return {
-    username: `@${accountName}`,
-    acct: `@${accountName}@${domainName}`
+    username: accountName,
+    acct: `${accountName}@${domainName}`,
```
Comment (h3poteto) [inline src/renderer/utils/tootParser.js]: Why did you remove the leading `@` ?
Pleroma's account name does not contain `@`, so I think that it can not compare here: https://github.com/h3poteto/whalebird-desktop/pull/780/files#diff-77909af62ad381cc7fda2de882cccd56R50

Comment (amyspark) [reply src/renderer/utils/tootParser.js]: Apologies, I forgot to revert this change.
